src/automod/Gems.cs

```diff
@@ -169,5 +182,13 @@ private async Task OnReactionAddedAsync(Cacheable<IUserMessage, ulong> message,
         // Post the message
         IUserMessage msg = await gemChannel.SendMessageAsync($"{gem} {textChannel.Mention} `{message.Id}` {gem}", embed: eb.Build()).ConfigureAwait(false);
         await msg.AddReactionAsync(reaction.Emote);
+
+        // Log contributors
+        await modlogChannel.SendMessageAsync(
+            $"Gem Reaction\n" +
+            $"Message by {FormatUtil.formatFullUser(newMessage.Author)} in {textChannel.Mention}\n" +
+            $"[Jump to Message]({newMessage.GetJumpUrl()})\n" +
+            $"Contributors: {rlist}"
+        );
```

Comment:
Use the existing logger
```suggestion
        DateTimeOffset now = DateTimeOffset.UtcNow;
        await Logger.Log(now, tc, WARN, $"A message in {textChannel.Mention} was gemmed by:\n\n{rlist}\n\n[Jump Link]({newMessage.GetJumpUrl()})", null);
```